istio/cluster-local-gateway/base/deployment.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: extensions/v1beta1
```

Comment:
Maybe Change it to v1 to support on Kubernetes 1.16?

Reply:
Sure, I'll make this change.

Reply:
For apps/v1, `spec.selector` is now required under the deployment spec. Can you also update it according to this [blog](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)? Thanks.